docs/coffee.ts

```diff
@@ -0,0 +1,273 @@
+const coffeeSelector = (document.getElementById("coffeeSelector") as HTMLInputElement);
+const teaSelector = (document.getElementById("teaSelector") as HTMLInputElement);
+const dessertSelector = (document.getElementById("dessertSelector") as HTMLInputElement);
+const menuGrid:any = document.getElementById("menuGrid");
+const menuButton = document.getElementById("coffeeMenu");
+const navModal = document.getElementById("navModal");
+let total:number;
+let products:any;
```

Comment:
Create a type if you are missing it for products.

Reply:
Created a type for products